lnclient/lnd/lnd.go

```diff
@@ -95,7 +95,18 @@ func (svc *LNDService) ListTransactions(ctx context.Context, from, until, limit,
 }
 
 func (svc *LNDService) GetInfo(ctx context.Context) (info *lnclient.NodeInfo, err error) {
-	resp, err := svc.client.GetInfo(ctx, &lnrpc.GetInfoRequest{})
+	if svc.nodeInfo == nil {
```

Comment:
this is set on startup, we don't need this code, right?

Reply:
Ah lol, true